docs/diagnostics/PX1022.md

```diff
@@ -0,0 +1,37 @@
+# PX1022
+This document describes the PX1022 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                                | Type  | Code Fix    | 
+| ------ | ---------------------------------------------------------------------------------------------------------------- | ----- | ----------- | 
+| PX1022 | The non-public graph extensions are not supported.                                                               | Error | Available   |
+
+## Diagnostic Description
+You can declare a class that is a graph extension. The class must be public.
```

Comment:
We need the explanation like:
"The non-public graph/DAC extensions are not loaded and wired up to graphs/DACs by platform" or
"The Acumatica Platform recognizes only public DAC/graph extensions"